src/Compilers/CSharp/Portable/Utilities/TypeSymbolExtensions.cs

```diff
@@ -133,6 +161,8 @@ public static bool HasCustomModifiers(this TypeSymbol type, bool flagNonDefaultA
             }
 
             return false;
+
+            bool checkTypeWithAnnotations(TypeWithAnnotations twa) => twa.CustomModifiers.Any() || twa.Type.HasCustomModifiers(flagNonDefaultArraySizesOrLowerBounds);
```

Comment:
>checkTypeWithAnnotations [](start = 17, length = 24)

nit: consider `static` and passing in `flagNonDefaultArraySizesOrLowerBounds` for clarity